hera_qm/tests/test_xrfi.py

```diff
@@ -4,12 +4,14 @@
 import numpy as np
 import pylab as plt
 import hera_qm.tests as qmtest
+from inspect import getargspec
+
 
 np.random.seed(0)
 
 SIZE = 100
 VERBOSE = False
-PLOT = False
+LOT = False
 
 FILES = {
```

Comment:
If possible, please delete this dict. It's only used in commented-out tests (see below). As it stands, these file locations don't even exist.